packages/cli/src/zosfiles/upload/dtu/DirToUSSDir.definition.ts

```diff
@@ -52,7 +52,8 @@ export const DirToUSSDirDefinition: ICommandDefinition = {
         UploadOptions.asciiFiles,
         UploadOptions.attributes,
         UploadOptions.maxConcurrentRequests,
-        UploadOptions.includeHidden
+        UploadOptions.includeHidden,
+        UploadOptions.encoding
     ],
```

Comment:
i like to include an example of using a new option that i added that way people know exactly how they can format their command when viewing the help. if others thumbs up would you mind adding an encoding example to the examples array?
```suggestion
          {
            description: strings.EXAMPLES.EX6, //youd have to edit this file too
            options: `"local_dir" "/a/ibmuser/my_dir" --encoding "IBM-1047"`
        }
```

Reply:
the suggested change correlates to lines 78-79 not 57-60. it was hard to indicate this on the extension i was using to create this review